microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/FaultToleranceCommand.java

```diff
@@ -422,6 +425,7 @@ private void updateMetricsAfter(Throwable throwable, boolean wasBreakerOpen, boo
             getCounter(method, BREAKER_CALLS_FAILED_TOTAL).inc();
             // If it will open, increment counter
             if (breakerWillOpen) {
+                System.out.println("### incrementing breaker");
```

Comment:
Do we want to remove this or change it to use a logger?

Reply:
We do, it slipped through the cracks. Change pushed now.